File: C++/976. Largest Perimeter Triangle.cpp
class Solution {
public:
    int largestPerimeter(vector<int>& n) {
        sort(n.rbegin(),n.rend());
        for(int i=0;i<n.size()-2;i++){
            if(n[i]<n[i+1]+n[i+2])
            return (n[i]+n[i+1]+n[i+2]);
        }
        return 0;

    }
};

File: JAVA/976. Largest Perimeter Triangle.java
class Solution {
    public int largestPerimeter(int[] n) {
        Arrays.sort(n);
        for(int i=n.length-1;i>=2;i--){
            if(n[i-1]+n[i-2]>n[i])
            return(n[i]+n[i-1]+n[i-2]);
        }
        return 0;
    }
}